components/print/cups/Makefile

```diff
@@ -54,37 +58,21 @@ CONFIGURE_OPTIONS +=	--with-smfmanifestdir=/lib/svc/manifest/application
 CONFIGURE_OPTIONS +=	--with-printcap=/etc/printers.conf
 CONFIGURE_OPTIONS +=	--with-cups-user=lp
 CONFIGURE_OPTIONS +=	--with-cups-group=lp
-CONFIGURE_OPTIONS +=	--with-php=no
+CONFIGURE_OPTIONS +=	--disable-systemd
 CONFIGURE_OPTIONS +=	--disable-static
 CONFIGURE_OPTIONS +=	--disable-gnutls
-CONFIGURE_OPTIONS +=	--disable-libusb
 CONFIGURE_OPTIONS +=	--enable-openssl
 CONFIGURE_OPTIONS +=	--enable-dbus
```

Comment:
--enable-openssl is not available anymore (gives a warning).
We should allow gnutls (remove --disable-gnutls) in order to support transport security.

Reply:
Done